src/main/java/com/flightstats/hub/dao/aws/S3Config.java

```diff
@@ -96,35 +97,35 @@ public void takeLeadership(Leadership leadership) {
             maxItemsEnforcer.updateMaxItems(configurations);
         }
 
+        private void updateRulesConfigForBucket(List<BucketLifecycleConfiguration.Rule> rules, String bucket) {
+            rules.addAll(getNonHubBucketLifecycleRules(bucket));
```

Comment:
Is this line going to cause `rules` to build up non-hub-created rules such that the second bucket that loops through will get both its non-hub rules AND the rules of the bucket(s) before it?

If so, the DR bucket is slowly going to build up copies of the normal bucket's static rules, e.g. 
1. run 1 will add x static rules from the hub (DR bucket: x static rules + hub rules)
2. run 2 will add x static rules from bucket + x static rules from run 1 (DR bucket: 2x static rules + hub rules)
3. run 3: DR bucket ends up with 3x static rules + hub rules

Reply:
good catch

Reply:
fixed